ush/parsing_namelists_CICE.sh

```diff
@@ -4,371 +4,594 @@ CICE_namelists(){
+   f_ardg         = 'x'
+   f_vrdg         = 'x'
+   f_dardg1dt     = 'x'
+   f_dardg2dt     = 'x'
+   f_dvirdgdt     = 'x'
+   f_opening      = 'x'
+   f_ardgn        = 'x'
+   f_vrdgn        = 'x'
+   f_dardg1ndt    = 'x'
+   f_dardg2ndt    = 'x'
+   f_dvirdgndt    = 'x'
+   f_krdgn        = 'x'
+   f_aparticn     = 'x'
+   f_aredistn     = 'x'
+   f_vredistn     = 'x'
+   f_araftn       = 'x'
+   f_vraftn       = 'x'
 /
 
 &icefields_pond_nml
-    f_apondn       = 'xxxxx'
-  , f_apeffn       = 'xxxxx'
-  , f_hpondn       = 'xxxxx'
-  , f_apond        = 'mdhxx'
-  , f_hpond        = 'mdhxx'
-  , f_ipond        = 'mdhxx'
-  , f_apeff        = 'mdhxx'
-  , f_apond_ai     = 'xxxxx'
-  , f_hpond_ai     = 'xxxxx'
-  , f_ipond_ai     = 'xxxxx'
-  , f_apeff_ai     = 'xxxxx'
+   f_apondn       = 'x'
+   f_apeffn       = 'x'
+   f_hpondn       = 'x'
+   f_apond        = 'mdhxx'
+   f_hpond        = 'mdhxx'
+   f_ipond        = 'mdhxx'
+   f_apeff        = 'mdhxx'
+   f_apond_ai     = 'x'
+   f_hpond_ai     = 'x'
+   f_ipond_ai     = 'x'
+   f_apeff_ai     = 'x'
 /
 
 &icefields_bgc_nml
-    f_faero_atm    = 'x'
-  , f_faero_ocn    = 'x'
-  , f_aero         = 'x'
-  , f_fNO          = 'x'
-  , f_fNO_ai       = 'x'
-  , f_fNH          = 'x'
-  , f_fNH_ai       = 'x'
-  , f_fN           = 'x'
-  , f_fN_ai        = 'x'
-  , f_fSil         = 'x'
-  , f_fSil_ai      = 'x'
-  , f_bgc_N_sk     = 'x'
-  , f_bgc_C_sk     = 'x'
-  , f_bgc_chl_sk   = 'x'
-  , f_bgc_Nit_sk   = 'x'
-  , f_bgc_Am_sk    = 'x'
-  , f_bgc_Sil_sk   = 'x'
-  , f_bgc_DMSPp_sk = 'x'
-  , f_bgc_DMSPd_sk = 'x'
-  , f_bgc_DMS_sk   = 'x'
-  , f_bgc_Nit_ml   = 'x'
-  , f_bgc_Am_ml    = 'x'
-  , f_bgc_Sil_ml   = 'x'  
-  , f_bgc_DMSP_ml  = 'x'
-  , f_bTin         = 'x'
-  , f_bphi         = 'x' 
-  , f_fbri         = 'x'    
-  , f_hbri         = 'x'
-  , f_grownet      = 'x'
-  , f_PPnet        = 'x'
+   f_faero_atm    = 'x'
+   f_faero_ocn    = 'x'
+   f_aero         = 'x'
+   f_fbio         = 'x'
+   f_fbio_ai      = 'x'
+   f_zaero        = 'x'
+   f_bgc_S        = 'x'
+   f_bgc_N        = 'x'
+   f_bgc_C        = 'x'
+   f_bgc_DOC      = 'x'
+   f_bgc_DIC      = 'x'
+   f_bgc_chl      = 'x'
+   f_bgc_Nit      = 'x'
+   f_bgc_Am       = 'x'
+   f_bgc_Sil      = 'x'
+   f_bgc_DMSPp    = 'x'
+   f_bgc_DMSPd    = 'x'
+   f_bgc_DMS      = 'x'
+   f_bgc_DON      = 'x'
+   f_bgc_Fe       = 'x'
+   f_bgc_hum      = 'x'
+   f_bgc_PON      = 'x'
+   f_bgc_ml       = 'x'
+   f_upNO         = 'x'
+   f_upNH         = 'x'
+   f_bTin         = 'x'
+   f_bphi         = 'x'
+   f_iDi          = 'x'
+   f_iki          = 'x'
+   f_fbri         = 'x'
+   f_hbri         = 'x'
+   f_zfswin       = 'x'
+   f_bionet       = 'x'
+   f_biosnow      = 'x'
+   f_grownet      = 'x'
+   f_PPnet        = 'x'
+   f_algalpeak    = 'x'
+   f_zbgc_frac    = 'x'
 /
 
 &icefields_drag_nml
-    f_drag         = 'mdhxx'
-  , f_Cdn_atm      = 'mdhxx'
-  , f_Cdn_ocn      = 'mdhxx'
+   f_drag         = 'mdhxx'
```

Comment:
These have also been removed (changed to 'x') in ufs-s2s (f_drag, f_Cdn_atm, f_Cdn_ocn).

Reply:
OK. I changed it.